cli/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "sqlness-cli"
+version = "0.1.0"
```

Comment:
how about keeping its version the same with the `sqlness` lib?

Reply:
OK, I will setup workspace to manage this.

Reply:
I plan to fix this in next PR since it will move lots of files, I will refactor project dir like this
```
sqlness
sqlness-cli
README.md
```